webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -94,6 +94,7 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
                 };
             }}/>
 
+    const changesTreeMessage = <p>Showing changes between <strong>{reference.id}</strong> and <strong>{compareReference.id}</strong></p>
```

Comment:
Why prep the message here? Can the internal component be smarter and understand whether or not to display a message?

Reply:
Messages are special to each component because they show different refs. I rather keep it this way